app/src/main/res/values-en/strings.xml

```diff
@@ -173,7 +173,7 @@ First off, understand that with this app, you make actual and direct contributio
 Anything you contribute in this app is directly added to the map, there is no third party in-between the app and OSM infrastructure.
 &lt;/p&gt;
 &lt;p&gt;
-Editing the OpenStreetMap anonymously is not possible, so any changes you make are made with your OSM user account. This means, that the date, content and implicitly the location of your changes are visible publicly on the openstreetmap website and are attributable to your account.
+Editing the OpenStreetMap anonymously is not possible, so any changes you make are made with your OSM user account. This means, that the date, content and the location of your changes are visible publicly on the openstreetmap website and are attributable to your account.
```

Comment:
@smichel17 's suggestion for comparison.
```suggestion
Editing OpenStreetMap anonymously is not possible. Any changes you make are publicly visible on the openstreetmap.org website and are attributed to your account. So are the date and location of those changes. Because StreetComplete should only be used for on-site survey, this implicitly reveals your location.
```

Reply:
My version was also intended only as cleanup for easier reading.

- Is OSM really referred to as "the OpenStreetMap"? Sounds strange to me. Like using the Git[Hub] for version control.
- "changes you make are made" is awkward, almost redundant. I fixed it with changing the sentence structure. Another alternative is, "so your changes are made".
- I like putting the website url there, gives a hint about where to look for that data.
- I slightly prefer my phrasing of "the changes are public, also this includes the date and location", but no strong feelings.
- I much prefer your phrase, "reveals where you have been".

Sorry for any verbosity, hard to edit on mobile!

Reply:
> My version was also intended only as cleanup for easier reading.

> * Is OSM really referred to as "the OpenStreetMap"? Sounds strange to me. Like using the Git[Hub] for version control.

Huh? The project is called "OpenStreetMap" (with capitalisation and all, see https://www.openstreetmap.org/about), and "OSM" is just an acronym for that name.

I don't get Git[Hub]/version control reference too - those are three different things:

- [version control](https://en.wikipedia.org/wiki/Version_control) is software which keeps history of editing of the files, so previous revisions can be retrieved, compared, merged etc. There are many different and incompatible types of them: CVS, SVN, Git, Mercurial, Bazaar, ...
- [Git](https://en.wikipedia.org/wiki/Git) is one specific type of version control software, written initially by Linus Torvalds, and gaining huge popularity in recent years.
- GitHub is (currently owned Microsoft) project management suite which incorporates (among other things) indexing engine, issue trackers, wiki software, e-mail interfacing, and both web-based and ssh-based Git version control software. So Git itself is only tiny part of GitHub.

> * "changes you make are made" is awkward, almost redundant. 

Agreed,  I prefer majority of @smichel17 version here.

> * I slightly prefer my phrasing of "the changes are public, also this includes the date and location", but no strong feelings.

Huh, I don't see that exact version anywhere?

> * I much prefer your phrase, "reveals where you have been".

I too like that part from @peternewman version.

So the combined version I'd prefer best:

> Editing OpenStreetMap anonymously is not possible. Any changes you make are publicly visible on the openstreetmap.org website and are attributed to your account. So are the date and location of those changes. Because StreetComplete should only be used for on-site survey, this implicitly reveals where you have been.

> Sorry for any verbosity, hard to edit on mobile!

Those PR nested comments are hard to follow/reply in readable way on desktop too :cry:

Reply:
> > * Is OSM really referred to as "the OpenStreetMap"? Sounds strange to me. Like using the Git[Hub] for version control.
> 
> Huh? The project is called "OpenStreetMap" (with capitalisation and all, see https://www.openstreetmap.org/about), and "OSM" is just an acronym for that name.
> 
> I don't get Git[Hub]/version control reference too - those are three different things:

I think @smichel17 was referring to the seemingly redundant "the" at the start, which you dropped in your reply comment. I think it should probably be "the openstreetmap.org website" or "OpenStreetMap" (but we could probably capitalise in the website name too). They may have a preferred style guide? Or not, they use with and without www on https://www.openstreetmap.org/copyright

> > * I slightly prefer my phrasing of "the changes are public, also this includes the date and location", but no strong feelings.
> 
> Huh, I don't see that exact version anywhere?

It's probably in the discussion thread, but not as a suggestion.
 
> I too like that part from @peternewman version.

I'm not sure I've actually added one have I, just added other people's as suggestions, as I'm about to do for @mnalis as I far prefer to concentrate on what's changed:

```suggestion
Editing OpenStreetMap anonymously is not possible. Any changes you make are publicly visible on the openstreetmap.org website and are attributed to your account. So are the date and location of those changes. Because StreetComplete should only be used for on-site survey, this implicitly reveals where you have been.
```

Finally, some minor tweaks on @mnalis suggestion to give my own one (but I wouldn't object to dropping "therefore" or my final change about are/have been):
```suggestion
Editing OpenStreetMap anonymously is not possible. Therefore any changes you make are publicly visible on the openstreetmap.org website and are attributed to your account; as are the date and location of those changes. Because StreetComplete should only be used for on-site survey, this implicitly reveals where you are or have been.
```